### PR TITLE
Position NED

### PR DESCRIPTION
**Pull request by aydal**
Git status output

modified: src/backend/src/plugins/telemetry/telemetry_service_impl.h
        modified: src/plugins/telemetry/include/plugins/telemetry/telemetry.h
        modified: src/plugins/telemetry/telemetry.cpp
        modified: src/plugins/telemetry/telemetry_impl.cpp
        modified: src/plugins/telemetry/telemetry_impl.h

### CODE REVIEW

**Comment by JonasVautherin** [comment]
You need to update the proto files and update the `proto/` git submodule in this repo. That is missing here. I believe that you made an update to the [`telemetry.proto`](https://github.com/mavlink/MAVSDK-Proto/blob/master/protos/telemetry/telemetry.proto#L9) file for python, didn't you?

So you need the same changes on MAVSDK-C++ (i.e. in this PR). Maybe you can open a PR to `MAVSDK-Proto` with your update `telemetry.proto`, too?

**Comment by aydal** [comment]
Removed 
src/backend/proto
src/third_party/

Were giving some trouble in pushing the commit

**Comment by JonasVautherin** [comment]
It seems like you are not up-to-date with `develop`, making it difficult to review (most of the commits are not part of this PR actually). Could you get your branch up-to-date, so that this PR only shows your changes? :blush:

**Comment by julianoes** [comment]
@aydal sorry it's impossible to review this. It seems like you added bunch of files. Please clean up the PR so it only shows the difff that you actually want to add.

**Comment by aydal** [comment]
@julianoes if the changes are okay then I can make one final push with the two deleted folders for easy integration.

File: src/backend/src/plugins/action/action_service_impl.h [hunk 1]
```diff
@@ -162,11 +162,18 @@ class ActionServiceImpl final : public rpc::action::ActionService::Service {
     grpc::Status
     SetTakeoffAltitude(grpc::ServerContext * /* context */,
                        const rpc::action::SetTakeoffAltitudeRequest *request,
-                       rpc::action::SetTakeoffAltitudeResponse * /* response */) override
+                       rpc::action::SetTakeoffAltitudeResponse *response) override
     {
         if (request != nullptr) {
             const auto requested_altitude = request->altitude();
-            _action.set_takeoff_altitude(requested_altitude);
+            mavsdk::Action::Result action_result = _action.set_takeoff_altitude(requested_altitude);
```
Comment (JonasVautherin) [inline src/backend/src/plugins/action/action_service_impl.h]: Yep! That's not related to `PositionNED`, but let's keep it in this PR for this time :+1:.

File: src/plugins/telemetry/include/plugins/telemetry/telemetry.h [hunk 2]
```diff
@@ -861,21 +861,15 @@ class Telemetry : public PluginBase {
  */
 bool operator==(const Telemetry::PositionVelocityNED &lhs,
                 const Telemetry::PositionVelocityNED &rhs);
-/**
- * @brief Equal operator to compare two `Telemetry::Position` objects.
- *
- * @return `true` if items are equal.
- */
 
 bool operator==(const Telemetry::PositionNED &lhs,
```
Comment (JonasVautherin) [inline src/plugins/telemetry/include/plugins/telemetry/telemetry.h]: You will probably need to document that, or the CI will complain :slightly_smiling_face:.

**Comment by JonasVautherin** [comment]
I just realized you were trying to merge `aydal:develop` into `mavlink:master`. You should merge your `develop` fork into the MAVSDK `develop` branch :slightly_smiling_face:. I changed it.

Still, Github says that this branch has conflicts that must be resolved. Meaning that you need to do something like `git rebase upstream/develop` into your branch. And you should not remove the submodules, if they are up-to-date they should just not be part of the diff.

**Comment by julianoes** [comment]
This pull request is actually not needed anymore because by now we have `PositionVelocityNED` which includes `PositionNED` which is added here.

Thanks anyway for your contribution @aydal.

**Pull request closed without merging**